### DISCUSSION TITLE
Problem getting cartridges to dump ROMS

### DISCUSSION THREAD

**Discussion by ddoyle11** [sanni/cartreader#459] [Need help]
Hi, I just got a cart reader from Save the Hero.  I followed the instructions, watched a few youtube videos and I just can't get this thing to work.  I put Yoshi's Cookie on the SNES port, selected the correct voltage. selected SNES, Read Rom.  Reset back to home screen, powered off, pulled out the microsd and the ROM isn't on there. I did some research before buying this and I'm stumped.  Any ideas?

**Comment by sanni** [comment]
Can you post the content of OSCR_LOG.txt from the SD card?

It should look like this:
````
OSCR HW3 V9.4


Searching database...
Found

Name: YOSHISCOOKIE
Type: LoROM SlowROM
ICs: ROM ONLY
ROM Size: 4Mbit
Banks: 16 Chips: 0
Sram Size: 0Kbit
ROM Version: 1.0
Checksum: 92FE
 
 


Saving to SNES/ROM/YOSHISCOOKIE/34/...
[*******************]
Chksum...OK
CRC32... EF15F4C3 -> Yoshi's Cookie (USA).sfc
````

Dumping should look like this:

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/SApt7RNwuzY/0.jpg)](https://www.youtube.com/watch?v=SApt7RNwuzY)

**Comment by SAVE-THE-HERO** [comment]
Hello.
I am the seller of your device.
If your device is broken, we can help.
Please email me at the email address on the card enclosed with the device.

But once this time we have received the advice from sanni-san, could you please try that?

**Reply by SAVE-THE-HERO** [reply]
I think your device's OS is v8.4.
The "content of OSCR_LOG.txt"  in sanni's advice will not came until you update your OS.

Testing has been done and the SD card is being shipped with a new SD card reader and SD card replacement to eliminate the sucked out ROM.
I therefore infer an SD card malfunction.

Check the version of your device and format the SD card for that version of data and write the data to it.
And pls try different SD card if you have.
The version can be selected from the menu screen under 'About'.
https://github.com/sanni/cartreader/wiki/Preparing-the-SD-card
https://github.com/sanni/cartreader/releases

 If you try formatting the SD card and it fails, try updating the device.
https://github.com/sanni/cartreader/wiki/How-to-flash-the-Arduino

I had same case happened twice.
One worked fine by formatting the SD card and the other was fixed by updating the device.

**Reply by ddoyle11** [reply]
Thanks for the information, looks like it's a bad sd card, I reformatted it and it doesn't work, I'll get a new card and give it a try
